src/Patch/Document.js

```diff
@@ -25,18 +26,7 @@ export default function(internals) {
      * @return {!Element}
      */
     function(localName) {
-      // Only create custom elements if this document is associated with the registry.
-      if (this.__CE_hasRegistry) {
-        const definition = internals.localNameToDefinition(localName);
-        if (definition) {
-          return new (definition.constructorFunction)();
-        }
-      }
-
-      const result = /** @type {!Element} */
-        (Native.Document_createElement.call(this, localName));
-      internals.patchElement(result);
-      return result;
+      return internals.createAnElement(this, localName, null);
```

Comment:
I like being able to reuse this but I wonder if there's a way to guarantee that the native `createElement` is called when the wrapper for `createElement` is called, rather than always using the native `createElementNS`.

Reply:
It looks like the template polyfill only works with `createElement`, making this necessary.

Reply:
Fixed.